gradle.properties

```diff
@@ -2,10 +2,13 @@
 bintrayVersion=1.4
 guavaVersion=16.0.1
 immutablesVersion=2.1.8
-jacksonVersion=2.4.4
+jacksonVersion=2.6.3
```

Comment:
Had trouble getting Dropwizard to work with conflicting deps, this is the 0.9.2 dropwizard Jackson version. Can probably force this back down to what it was...will iterate.